languages/python/exercises/concept/tuples/.docs/introducton.md

```diff
@@ -0,0 +1,113 @@
+In Python, a [tuple](https://docs.python.org/3/library/stdtypes.html#tuple) is an immutable collection of items in _sequence_. Like most collections, tuples can hold any (or multiple) data type(s) -- including other tuples. Like any sequence, items are referenced by 0-based index number. Tuples support all [common sequence operations](https://docs.python.org/3/library/stdtypes.html#common-sequence-operations).
+
+## Tuple Construction
+
+Tuples can be formed in multiple ways, using either the `tuple` class constructor or the `tuple` literal declaration.
+
+#### Using the `tuple()` constructor:
+
+```python
+no_elements = tuple()
+no_elements
+>>> ()
+
+#elements are iterated through and added to the tuple in order
+multiple_elements_string = tuple("Timbuktu")
+multiple_elements_string
+>>> ('T', 'i', 'm', 'b', 'u', 'k', 't', 'u')
+
+multiple_elements_list = tuple(["Parrot", "Bird", 334782])
+multiple_elements_list
+>>> ("Parrot", "Bird", 334782)
+
+```
+
+#### Declaring a tuple _literal_ :
+
+Note: generally parentheses are not _required_ to create a `tuple` literal - only commas. Parentheses are only required in cases of ambiguity.
+
+```python
+no_elements = ()
+no_elements
+>>> ()
+
+
+elements_separated_with_commas = "Parrot", "Bird", 334782
+elements_separated_with_commas
+>>> ("Parrot", "Bird", 334782)
+
+elements_with_commas_and_parentheses = ("Triangle", (60, 60, 60))
+elements_with_commas_and_parentheses
+>>> ("Triangle", (60, 60, 60))
+
+```
+
+## Concatenation
+
+Tuples can be _concatenated_. Using the plus **`+`** operator unpacks each tuple and creates a new tuple from the combined elements.
+
+```python
+new_via_concatenate = ("George", 5) + ("cat", "Tabby")
+new_via_concatenate
+>>> ("George", 5, "cat", "Tabby")
+
+new_nested_concatenate = ("Triangle", (60, 60, 60)) + ('orange', 'dashed-outline')
+new_nested_concatenate
+>>> ('Triangle', (60, 60, 60), 'orange', 'dashed-outline')
```

Comment:
Maybe just have one example?

Reply:
Agreed - especially since there is no task emphasizing that **+** only unpacks one layer.  Removing second example.